flake/dev/fmt.nix

```
@@ -18,6 +18,10 @@
           nixf-diagnose = {
             enable = true;
             priority = -1;
+            excludes = [
+              # Warns against using the `path` argument, but we cannot really workaround it.
```

Maybe something like this?

OTTOMH, I don't know the actual wording of the diagnostic hint...


```suggestion
              # Warns that the `path` argument could be confused with `builtins.path`.
```

---

We could consider disabling the "argument shadows builtin" diagnostic instead of excluding this file. Up to you.